COLLABORATOR_GUIDE.md

```diff
@@ -111,10 +111,9 @@ comment that explains why the PR does not require a CI run.
 
 ### Code Reviews
 
-All pull requests must be reviewed and accepted by a Collaborator with
-sufficient expertise who is able to take full responsibility for the
-change. In the case of pull requests proposed by an existing
-Collaborator, an additional Collaborator is required for sign-off.
+All pull requests must be reviewed and accepted by two Collaborators with
```

Comment:
This may be a good opportunity to clarify that we mean *at least* two, and not exactly two.